### DISCUSSION TITLE
Using immutable classes

### DISCUSSION THREAD

**Discussion by derolf** [isar/isar#78] [Help]
Right now, `isar` requires `nullable` or `late` fields. Is it possible to combine `isar` with `freezed?`

**Comment by om-ha** [comment]
Silly me! You can use `nullable` types with TypeConverter, you just need to make sure the TypeConverter works on nullable types (generic definition, and from/to Isar overridden methods)

~~It looks like `nullable` fields don't work when using TypeConverter annotation ([TypeConverter documentation](https://isar.dev/type-converters))~~
